### PR TITLE
don't prompt to sell starter licenses in China

### PR DESCRIPTION
**Pull request by smallst**

### CODE REVIEW

File: compile-static-templates.coffee [hunk 1]
```diff
@@ -48,6 +48,7 @@ compile = (contents, locals, filename, cb) ->
     locals.me.useDexecure = -> not (locals.chinaInfra ? false)
     locals.me.useSocialSignOn = -> not (locals.chinaInfra ? false)
     locals.me.useGoogleAnalytics = -> not (locals.chinaInfra ? false)
+    locals.me.useStripe = -> not (locals.chinaInfra ? false)
```
Comment (codeluggage) [inline compile-static-templates.coffee]: Nice addition! This will be useful.

File: app/models/User.coffee [hunk 2]
```diff
@@ -619,6 +619,7 @@ module.exports = class User extends CocoModel
   hideDiplomatModal: -> features?.china ? false
   showChinaRemindToast: -> features?.china ? false
   showOpenResourceLink: -> not (features?.china ? false)
+  shouldUpsell: -> (not (features?.china ? false)) and (@get('preferredLanguage') isnt 'nl-BE')
```
Comment (codeluggage) [inline app/models/User.coffee]: Why are we checking for the Dutch language code here?

Comment (wjllance) [reply app/models/User.coffee]: That's the old logic?

**Comment by wjllance** [comment]
> Have a question about why we are checking for the Dutch language code, but otherwise this looks good!
> 
> The conditionals in coffeescript are difficult for me, so it's a good idea to double check the double negative check in the `shouldUpsell` function :)
> 
> Do you have any thoughts on why these tests failed? I don't quite see why they would fail. If you don't know why, then let's just make them run again to see if it was random.
> 
> ```
> .................................................
> 
> ERROR: 'No new students imported. Error:', [Object{isError: true, errorID: 'google-id-exists', error: Object{_id: ..., permissions: ..., email: ..., anonymous: ..., name: ..., gplusID: ..., firstName: ..., lastName: ..., role: ...}}, Object{isError: true, errorID: 'google-id-exists', error: Object{_id: ..., permissions: ..., email: ..., anonymous: ..., name: ..., gplusID: ..., firstName: ..., lastName: ..., role: ...}}]
> 
> ......................................
> 
> Firefox 66.0.0 (Linux 0.0.0) Client Check keys when language is English with the letters jumbled should have the same keys in each entry as in English FAILED
> 
> 	TypeError: en[key] is undefined in public/javascripts/test.js (line 575383)
> 
> 	@public/javascripts/test.js:575383:15
> 
> Firefox 66.0.0 (Linux 0.0.0) Client Check keys when language is English with the letters jumbled should have the same keys in each entry as in English FAILED
> 
> 	TypeError: en[key] is undefined in public/javascripts/test.js (line 575383)
> 
> 	@public/javascripts/test.js:575383:15
> 
> ............................................................
> ```

Oh, I think this one is due to the last one revert commit on master. Sorry I didn't wait for all test finish then merged it... Already raised a new commit to solve that

File: app/models/User.coffee (hunk 2 above)
Comment (smallst) [reply app/models/User.coffee]: old logic. You can find it at EnrollmentView#line116 in old file

Comment (codeluggage) [reply app/models/User.coffee]: Ah of course, I missed that!

File: app/models/User.coffee [hunk 3]
```diff
@@ -619,8 +619,9 @@ module.exports = class User extends CocoModel
   hideDiplomatModal: -> features?.china ? false
   showChinaRemindToast: -> features?.china ? false
   showOpenResourceLink: -> not (features?.china ? false)
+  useStripe: -> (not ((features?.china ? false) or (features?.chinaInfra ? false))) and (@get('preferredLanguage') isnt 'nl-BE')
 
-  # Ozaria flags
+# Ozaria flags
```
Comment (codeluggage) [inline app/models/User.coffee]: Is this comment on the wrong indent?

**Comment by wjllance** [comment]
seems has some other test error.. will have a look tomorrow

**Pull request merged**